commands/operator-sdk/cmd/test/local.go

```diff
@@ -141,6 +149,11 @@ func testLocalFunc(cmd *cobra.Command, args []string) {
 			}
 		}()
 	}
+	if tlConfig.image != "" {
+		if err := replaceImage(tlConfig.namespacedManPath, tlConfig.image); err != nil {
```

Comment:
Any reason we can't replace the image before combining the manifests, i.e. [here](https://github.com/operator-framework/operator-sdk/blob/master/commands/operator-sdk/cmd/test/local.go#L88-L91)?

Reply:
We can't do that because a user can pass in their own namespaced manifest, in which case we would skip the combination step.